src/Talk/Create/TalkFactory.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Talk\Create;
+
+use App\Entity\Speaker;
+use App\Talk\Talk;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+class TalkFactory
+{
+    public function createFromRequest(CreateTalkRequest $createTalkRequest): Talk
+    {
+        return new Talk(
+            $this->nextIdentity(),
+            $createTalkRequest->title,
+            $createTalkRequest->description,
+            $createTalkRequest->speaker
+        );
+    }
+
+    public function createWith(string $title, string $description, Speaker $speaker): Talk
+    {
+        return new Talk(
+            $this->nextIdentity(),
+            $title,
+            $description,
+            $speaker
+        );
+    }
+
+    private function nextIdentity(): UuidInterface
```

Comment:
This should be a generator class passed in. I assume you kept it that way from other implementations? Then should we do another issue for that?